packages/docregistry/src/context.ts

```diff
@@ -208,80 +208,46 @@ class Context<T extends DocumentRegistry.IModel> implements DocumentRegistry.ICo
   readonly urlResolver: IRenderMime.IResolver;
 
   /**
-   * Populate the contents of the model, either from
-   * disk or from the modelDB backend.
+   * Initialize the context.
    *
-   * @returns a promise that resolves upon model population.
+   * @param isNew - Whether it is a new file.
+   *
+   * @returns a promise that resolves upon initialization.
    */
-  fromStore(): Promise<void> {
+  initialize(isNew: boolean): Promise<void> {
+    if (isNew) {
```

Comment:
I just verified that this change still works with the `modelDB` in the Google Realtime stuff. I know that's going away soon, but wanted to make sure that we don't do it accidentally.